attributes/default.rb

```diff
@@ -3,8 +3,8 @@
 default[:redmine][:home]         = '/home/redmine'
 default[:redmine][:host]         = 'redmine.example.com'
 default[:redmine][:listen_port]  = 80
-default[:redmine][:version]      = '2.6.1'
-default[:redmine][:ruby_version] = '1.9.3-p484'
+default[:redmine][:version]      = '3.2.0'
+default[:redmine][:ruby_version] = '2.3.0'
```

Comment:
Just to be on the safe side, I suppose that redmine version plays well with that ruby version?